packages/surveys/src/components/wrappers/auto-close-wrapper.tsx

```
@@ -62,13 +62,17 @@ export function AutoCloseWrapper({
   }, [survey.autoClose]);
 
   return (
-    <div className="fb-h-full fb-w-full">
+    <div className="fb-h-full fb-w-full fb-flex fb-flex-col">
+      <div
+        onClick={stopCountdown}
+        onMouseOver={stopCountdown}
+        className="fb-h-full fb-w-full fb-pb-2"
+        data-testid="fb__surveys__auto-close-wrapper">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)